examples/hil.cfg

```diff
@@ -90,7 +93,7 @@ uri = sqlite:////absolute/path/to/hil.db
 #dry_run=
 
 [network-daemon]
-# The amount of time in seconds to sleep after attempting to empty the journal 
+# The amount of time in seconds to sleep after attempting to empty the journal
```

Comment:
whitespace removed by my text editor.

Reply:
See also #873 (though obviously doesn't need to be dealt with here).